pkg/config/config.go

```diff
@@ -55,6 +56,7 @@ type Config struct {
 	LogLevel                 string        `yaml:"logLevel"`
 	EnabledPlugin            []string      `yaml:"enabledPlugin"`
 	MetricsInterval          time.Duration `yaml:"metricsInterval"`
```

Comment:
go ahead and un-export this field and throw a `//deprecated` on it so it doesn't get used anymore